cmd/root.go

```diff
@@ -327,20 +327,28 @@ func InitConfig(ctx context.Context, clients *shared.ClientFactory, rootCmd *cob
 // It listens for process interrupts and sends to IOStreams' GetInterruptChannel() for use in
 // in communicating process interrupts elsewhere in the code.
 func ExecuteContext(ctx context.Context, rootCmd *cobra.Command, clients *shared.ClientFactory) {
+	// Derive a cancel context that is cancelled when the main execution is interrupted or cleaned up.
+	// Sub-commands can register for the cleanup wait group with clients.CleanupWaitGroup.Add(1)
+	// and listen for <-ctx.Done() to be notified when the main execution is terminated and have
```

Comment:
Awesome language choice here 📚 ✨ 

I was going to suggest "interrupted" instead of "terminated" without realizing that the deferred `cancel` below is so useful for whatever goroutine cleanup might exist!

Reply:
I think "interrupted" is a good choice and more general than "terminated." Commit 68ecf29 updates the wording to be interrupted!

Reply:
Ooh I'm alright with either! Interrupted might be confused with an actual `^C` but the surrounding wording I think is helpful so no worries with either choice 🤓